### DISCUSSION TITLE
Loading Wallet

### DISCUSSION THREAD

**Discussion by kabamas26** [Chia-Network/chia-blockchain#4083] [Support]
Hello all!!

I made a mistake setting the small SDD(**250GB**) that I have instead of SDD **1TB** ... yea I know :-s ... the thing is that I deleted all the plots from the small one ... I try to get into the Chia System and I cant ... it shows me a message **loading wallet** ... I was waiting all night long and nothing ... I deleted the Chia System and installed again and the same error ... somebody can help me please?

Thanks in advance!

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
